src/analysis/schwartz.lean

```diff
@@ -0,0 +1,400 @@
+/-
```

Comment:
Maybe rename the file to `schwartz_space`? There are many things that could be named Schwartz...